src/ui/locales/uk.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Показувати атрибуцію",
```

Comment:
```suggestion
    "AttributionControl.ToggleAttribution": "Показати атрибуцію",
```
based on https://github.com/mapbox/mapbox-gl-js/pull/11733